.eslintrc

```diff
@@ -13,4 +13,4 @@
     "strict": ["error", "global"],
     "curly": "warn"
   }
-}
+}
```

Comment:
You have remove last line in file? What for?

Reply:
It was removed by mistake, I turned it back,